interface-definitions/interfaces_wireless.xml.in

```diff
@@ -759,9 +768,10 @@
                     </properties>
                     <defaultValue>wpa+wpa2</defaultValue>
                   </leafNode>
+                  #include <include/generic-username.xml.i>
                   <leafNode name="passphrase">
                     <properties>
-                      <help>WPA personal shared pass phrase. If you are using special characters in the WPA passphrase then single quotes are required.</help>
+                      <help>WPA personal shared passphrase. If you are using special characters in the WPA passphrase then single quotes are required.</help>
```

Comment:
`<help>WPA passphrase to be used. WPA personal mode will be used if username is not specified, enterprise mode will be used otherwise.</help>`

Reply:
Conditional checks are not possible indeed. Do you think that the used regex (min 8 max 63) characters will not work for enterprise environments?

Reply:
Then please alter your proposed CLI as outlined in https://github.com/vyos/vyos-1x/pull/3711#issuecomment-2186035530, many thanks